[2023-10-03T13:37:27.411+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: capstone_project.write_parquet_to_bq scheduled__2023-09-27T00:00:00+00:00 [queued]>
[2023-10-03T13:37:27.420+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: capstone_project.write_parquet_to_bq scheduled__2023-09-27T00:00:00+00:00 [queued]>
[2023-10-03T13:37:27.420+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2023-10-03T13:37:27.437+0000] {taskinstance.py:1350} INFO - Executing <Task(GCSToBigQueryOperator): write_parquet_to_bq> on 2023-09-27 00:00:00+00:00
[2023-10-03T13:37:27.443+0000] {standard_task_runner.py:57} INFO - Started process 984 to run task
[2023-10-03T13:37:27.446+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'capstone_project', 'write_parquet_to_bq', 'scheduled__2023-09-27T00:00:00+00:00', '--job-id', '350', '--raw', '--subdir', 'DAGS_FOLDER/capstone_project.py', '--cfg-path', '/tmp/tmpaq1ik4ii']
[2023-10-03T13:37:27.447+0000] {standard_task_runner.py:85} INFO - Job 350: Subtask write_parquet_to_bq
[2023-10-03T13:37:27.491+0000] {task_command.py:410} INFO - Running <TaskInstance: capstone_project.write_parquet_to_bq scheduled__2023-09-27T00:00:00+00:00 [running]> on host 9a30478b5169
[2023-10-03T13:37:27.573+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='capstone_project' AIRFLOW_CTX_TASK_ID='write_parquet_to_bq' AIRFLOW_CTX_EXECUTION_DATE='2023-09-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-27T00:00:00+00:00'
[2023-10-03T13:37:27.580+0000] {base.py:73} INFO - Using connection ID 'google_cloud_conn' for task execution.
[2023-10-03T13:37:27.586+0000] {gcs_to_bigquery.py:372} INFO - Using existing BigQuery table for storing data...
[2023-10-03T13:37:27.639+0000] {bigquery.py:2280} INFO - Project is not included in destination_project_dataset_table: {maurits_dataset}.{rocket_launches}; using project "aflow-training-rabo-2023-10-02"
[2023-10-03T13:37:27.640+0000] {gcs_to_bigquery.py:376} INFO - Executing: {'load': {'autodetect': True, 'createDisposition': 'CREATE_IF_NEEDED', 'destinationTable': {'projectId': 'aflow-training-rabo-2023-10-02', 'datasetId': '{maurits_dataset}', 'tableId': '{rocket_launches}'}, 'sourceFormat': 'PARQUET', 'sourceUris': ['gs://aflow-training-rabo-2023-10-02/maurits_dataset/2023-09-27.parquet'], 'writeDisposition': 'WRITE_APPEND', 'ignoreUnknownValues': False}}
[2023-10-03T13:37:27.640+0000] {bigquery.py:1562} INFO - Inserting job ***_capstone_project_write_parquet_to_bq_2023_09_27T00_00_00_00_00_b2f5dcadaf187d9484736ea837a98eec
[2023-10-03T13:37:28.124+0000] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py", line 377, in execute
    job = self._submit_job(self.hook, job_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py", line 306, in _submit_job
    nowait=True,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 468, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1565, in insert_job
    job._begin()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 518, in _begin
    timeout=timeout,
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/client.py", line 782, in _call_api
    return call()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/retry.py", line 288, in retry_wrapped_func
    on_error=on_error,
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/retry.py", line 190, in retry_target
    return target()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/_http/__init__.py", line 494, in api_request
    raise exceptions.from_http_response(response)
google.api_core.exceptions.NotFound: 404 POST https://bigquery.googleapis.com/bigquery/v2/projects/aflow-training-rabo-2023-10-02/jobs?prettyPrint=false: Not found: Dataset aflow-training-rabo-2023-10-02:{maurits_dataset}
[2023-10-03T13:37:28.181+0000] {taskinstance.py:1373} INFO - Marking task as FAILED. dag_id=capstone_project, task_id=write_parquet_to_bq, execution_date=20230927T000000, start_date=20231003T133727, end_date=20231003T133728
[2023-10-03T13:37:28.202+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 350 for task write_parquet_to_bq (404 POST https://bigquery.googleapis.com/bigquery/v2/projects/aflow-training-rabo-2023-10-02/jobs?prettyPrint=false: Not found: Dataset aflow-training-rabo-2023-10-02:{maurits_dataset}; 984)
[2023-10-03T13:37:28.222+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 1
[2023-10-03T13:37:28.245+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
